### PR TITLE
Remove all-caps default in /tip (yge)

### PR DESCRIPTION
**Pull request by bumi**
##### Description
Similar to #116 and discussed in #151 this PR removes the global (on body) `text-transform: uppercase` style from the /tip (yge) application. 

As discussed in #151 properties like the bounty status should also be normalized (capitalized in the UI) 
These properties are stored in the DB and we need to decide where to put that presentation logic. 

##### Checklist

- [ ] normalize bounty properties (status, etc.)

##### Affected core subsystem(s)
ui

##### Refers/Fixes
* #151 - "typography cleanup after PR #116"

### CODE REVIEW

**Comment by owocki** [comment]
is it just me, or would it be good if headers were still uppercase?  

screenshots follow 

screencapture-localhost-8080-tip-send-2-1514753821341
screencapture-localhost-8080-tip-send-2-1514753839333
screencapture-localhost-8080-tip-1514753877701
screencapture-localhost-8080-tip-1514753882162

**Comment by bumi** [comment]
we somehow should unify the different stylesheets that are used.
The /explorer has a special h1-h6 style for that (https://github.com/gitcoinco/web/blob/master/app/assets/v2/css/gitcoin.css#L31) but that stylesheet is not used in /tip.
Unifying the base stylesheets would make it easier. 
Sadly I am not a frontend developer, any ideas on how to best approach that?

**Comment by owocki** [comment]
awesome, thank you :)

File: app/dashboard/templates/yge/send1.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
         </span>
 		<h1>Send Tip ⚡</h1>
 		<h3>It's Fast.  It's Easy.  It's Free.️</h3>
-        <h4>(Supports any github username or email address)</h4>
+        <p>(Supports any github username or email address)</p
```
Comment (bumi) [inline app/dashboard/templates/yge/send1.html]: oh nooo... I commited broken HTML. 
As this PR is already merged, this is fixed in: https://github.com/gitcoinco/web/pull/202/commits/a6a4c7d8ec8e9c10c60da3edc4aff2ec38e8c5e1#diff-88a1050efecc8245bddec63cb35b9412R24

BUT as that other PR is kinda big, shall I do a quick PR to fix only this error, that we easily can merge?

**Comment by owocki** [comment]
accidental bounty.  claiming it myself now

**Comment by bumi** [comment]
any idea how we should approach the bounty status text normalization? 
It seems there is some stuff in python and some stuff in some JS.

**Comment by owocki** [comment]
sorry, im not sure i follow the question.

**Comment by bumi** [comment]
I was referring to the second topic in https://github.com/gitcoinco/web/issues/151

**Pull request merged**